rules/macos/persistence_finder_sync_plugin_pluginkit.toml

```
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/12/18"
+maturity = "production"
+updated_date = "2020/12/18"
+
+[rule]
+author = ["Elastic"]
+description = """
+Finder Sync plugins enables users to extend Finder’s functionality by modifying the user interface. Adversaries may
+abuse this feature by adding a rogue Finder Plugin to repeatedly execute malicious payloads as part of persistence.
+"""
+false_positives = ["Trusted Finder Sync Plugins"]
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Finder Sync Plugin Registered and Enabled"
+references = [
+    "https://github.com/specterops/presentations/raw/master/Leo Pitt/Hey_Im_Still_in_Here_Modern_macOS_Persistence_SO-CON2020.pdf",
+]
+risk_score = 47
+rule_id = "37f638ea-909d-4f94-9248-edd21e4a9906"
+severity = "medium"
+tags = ["Elastic", "Host", "macOS", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan=5s
```

To help reduce scope

```suggestion
sequence by host.id, user.id with maxspan=5s
```